test/unittest/xmlparser/wrapper/XMLParserTest.hpp

```diff
@@ -252,10 +252,18 @@ class XMLParserTest : public XMLParser
 
     static XMLP_ret getXMLThroughputController_wrapper(
             tinyxml2::XMLElement* elem,
-            ThroughputControllerDescriptor& throughputController,
+            ThroughputControllerDescriptor& throughtController,
```

Comment:
I think this was not intended to be changed
```suggestion
            ThroughputControllerDescriptor& throughtputController,
```